bin/collapse-filters.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const fs = require('fs');
+const path = require('path');
+const argv = require('minimist')(process.argv.slice(2));
+const styleToFilters = require('../lib/styleToFilters.js');
+
+const usage = `usage:
+
+  collapse-filters [args]
```

Comment:
Open to other name suggestions. Others I considered were:

- filters-for-sources (too many dashes)
- parse-filters (not really as simple as parsing, since this logic collapses filters and extracts attributes and zoom levels)

Reply:
`vtshaver-filters`?

Reply:
Ooo, I like it. I renamed to `vtshaver-filters`